xml/art_solution_automation.xml

```diff
@@ -0,0 +1,416 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="file:/usr/share/xml/geekodoc/rng/geekodoc5-flat.rnc"
+   type="application/relax-ng-compact-syntax"?>
+
+<!DOCTYPE article
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+<!--
+Confluence page:
+https://confluence.suse.com/x/8IEnGw
+-->
+
+<article version="5.1" xml:lang="en" xml:id="art-sol-automation"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>SAP Automation</title>
+  <info>
+    <abstract>
+      <para>
+        This article offers some automation solutions for &sap; administrators
+        to use with &salt; to efficiently configure &ha; and &sap; systems.
+        The solutions that is described here works from &slsreg;&nbsp;12
+        and beyond. All packages are available in the SLES4SAP module.
+      </para>
+    </abstract>
+  </info>
+
+  <note>
+    <title>No Official &suse; Support for &terraform;</title>
```

Comment:
I guess we still need to remove `terraform` from here

Reply:
Ok, removed.